src/views/Update/index.js

```diff
@@ -0,0 +1,19 @@
+import { useUser } from 'hooks';
+import GordonUnauthorized from 'components/GordonUnauthorized';
+import GordonLoader from 'components/Loader';
+import { UpdatePage } from './UpdatePage';
+
+/**
+ * Sends an update form to the development office
+ */
+
+const Update = (props) => {
+  const { profile, loading } = useUser();
+  return loading
+    ? <GordonLoader />
+    : profile
+      ? <UpdatePage profile={profile} authentication={props.authentication} />
+       :<GordonUnauthorized feature={'the Alumni Update Page'} />
```

Comment:
This is one case where `if` statements are probably much more readable than nested ternary operators.